Resources/config/authorize.xml

```diff
@@ -3,11 +3,14 @@
 <container xmlns="http://symfony.com/schema/dic/services"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
-
+	
+	<parameters>
+	  <parameter key='fos_oauth_server.authorize.form.class'>FOS\OAuthServerBundle\Form\Type\AuthorizeFormType</parameter> 
+	</parameters>
     <services>
         <service id="fos_oauth_server.authorize.form" class="Symfony\Component\Form\Form">
             <argument>%fos_oauth_server.authorize.form.name%</argument>
-            <argument>%fos_oauth_server.authorize.form.type%</argument>
+            <argument>%fos_oauth_server.authorize.form.class%</argument>
```

Comment:
It seems, the createName can take the type or class name. are you sure it will break BC?

````
ver: 2.3 

FormInterface	createNamed(string|integer $name, string|FormTypeInterface $type = 'form', mixed $data = null, array $options = array())
Returns a form.
```